kotlin/build.gradle

```diff
@@ -75,6 +76,7 @@ buildscript {
               'common': "org.jetbrains.kotlin:kotlin-test-common",
               'annotations': "org.jetbrains.kotlin:kotlin-test-annotations-common",
               'jdk': "org.jetbrains.kotlin:kotlin-test-junit",
+              'kotlintest': "io.kotlintest:kotlintest-runner-junit5:3.3.3",
```

Comment:
Are there no common modules we can use for `workflow-core`, `workflow-runtime`, etc?

Reply:
Nope. It builds strictly on Junit5.